packages/cli/src/commands/init/printRunInstructions.ts

```diff
@@ -34,6 +34,13 @@ function printRunInstructions(projectDir: string, projectName: string) {
   ${chalk.green(`Run instructions for ${chalk.bold('Android')}`)}:
     • Have an Android emulator running (quickest way to get started), or a device connected.
     • cd "${projectDir}" && npx react-native run-android
+
+  ${chalk.magenta(
+    `Run instructions for ${chalk.bold('Windows')} and ${chalk.bold('macOS')}`,
+  )}:
+    • See ${chalk.underline(
+      'http://aka.ms/ReactNative',
```

Comment:
I prefer full URL instead of using a shortener, WDYT?
```suggestion
      'https://microsoft.github.io/react-native-windows/',
```
Or at least use `https://` :) I'm fine with both

Reply:
I think we would want to be able to update the target URL or title of the project without requiring updating the CLI

Reply:
As long as it's controlled from someone from Microsoft, then I'm good. Just https then please :D

Reply:
Yep I'm on the RN team at Microsoft :) and yes can do https

Reply:
👍